src/mlpack/tests/layer_names_test.cpp

```diff
@@ -52,6 +52,14 @@ BOOST_AUTO_TEST_CASE(LayerNameVisitorTest)
   LayerTypes<> pReLU = new PReLU<>();
   LayerTypes<> sigmoidLayer = new SigmoidLayer<>();
   LayerTypes<> logSoftMax = new LogSoftMax<>();
+  LayerTypes<> lstmLayer = new LSTM<>(100, 10);
+  LayerTypes<> creluLayer = new CReLU<>();
+  LayerTypes<> highwayLayer = new Highway<>();
+  LayerTypes<> gruLayer = new GRU<>();
+  LayerTypes<> glimpseLayer = new Glimpse<>();
+  LayerTypes<> fastlstmLayer = new FastLSTM<>();
+  LayerTypes<> weightnormLayer = new WeightNorm<>();
```

Comment:
@khizirsiddiqui, If you pass an Identity layer as an argument to `weightNormLayer` while constructing then I think The tests will pass. If you would have observed `WeightNorm` code you would have seen that it needs a layer for `construction` if you will not pass any layer then it will lead to `deleting` a non-existent memory thus, memory access violation. This is just a guess. Let me know if it is incorrect.

Reply:
Aaah, silly me. Yeah, it passed the test, looking for the CI tests.